addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -185,7 +185,7 @@
     </t>
     <t t-name="point_of_sale.OrderLinesReceipt">
         <div id="receipt-orderline" t-foreach="receiptData.receipt.orderlines" t-as="line" t-key="line.id" t-att-class="{'border-start border-3 ps-1': line.isPartOfCombo }">
-            <div t-esc="line.product_name_wrapped[0]" />
+            <div t-esc="line.product_name_wrapped[0]">
```

Comment:
Hi, @caburj 
From the structure of the xml, I think the `/` is the redundant one
But the tour test expects the `<div>` selfclosed here.
Could you help me confirm which one is correct?